tests/test_humidity_sensor.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from greenpithumb import humidity_sensor
+from greenpithumb import dht11_exceptions
+
+
+class HumiditySensorTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_dht11_result = mock.Mock()
+        self.humidity_sensor = humidity_sensor.HumiditySensor(mock.Mock(
+            return_value=self.mock_dht11_result))
```

Comment:
>I'm having trouble getting this to work. It appears that with the way you have it set up above the error code and humidity aren't getting assigned values

Whoops, I got this wrong. We want `self.mock_dht11_result` to be the return value from the `read` method, not the return value from the object itself. Created a PR you can pull into this branch to fix it:

https://github.com/JeetShetty/GreenPiThumb/pull/16